packages/mongo/mongo_driver.js

```diff
@@ -133,12 +139,17 @@ MongoConnection = function (url, options) {
   self._observeMultiplexers = {};
   self._onFailoverHook = new Hook;
 
+  const userOptions = {
+    ...(Mongo._connectionOptions || {}),
+    ...(Meteor.settings && Meteor.settings.overrideMongoOptions || {})
```

Comment:
I think all Meteor specific settings should be under a specific Meteor key, something like Galaxy has  `"galaxy.meteor.com"` for its env variables. So I'm thinking `Meteor.settings.meteor.mongoOptions`.

Reply:
Great.